recipes/pymediainfo/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "pymediainfo" %}
+{% set version = "4.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 392d99d6bf74046ebaa2f7036d92d5327611d27532a384540e9310a62b8be26d
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools_scm
+  run:
+    - python
+    - setuptools
```

Comment:
This is a weird requirement, but it the program is using pkg_resources during runtime to discover its `__version__`.